config/inovelli/nzw31.xml

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!-- Inovelli Dimmer NZW31 -->
 <Product xmlns="http://code.google.com/p/open-zwave/">
   <!-- Configuration Parameters -->
   <CommandClass id="112">
-    <Value genre="config" index="1" label="Dimming/Brightening Step Level" min="1" max="99" value="1" size="1" type="byte">
-      <Help>Each step of the dimming process will increase or decrease the brightness by this percentage. Increasing this will make the transition time quicker.</Help>
+    <Value genre="config" index="1" label="Dimming/Brightening Step Level" min="1" max="99" units="%" value="1" size="1" type="byte">
```

Comment:
On my NZW31 (type `1f01`, id `1f01`), parameter 1 can be set to 0 to disable dimming (works like on/off).

Reply:
Hi there, just to confirm. There was an April firmware change that allows for a setting of 0 for parameter 1. This indeed causes the dimmer to instantly turn on/off like a typically switch. 

Eric M
VP Technology | Inovelli